spec/controllers/service_requests/post_feedback_spec.rb

```diff
@@ -1,85 +0,0 @@
-# Copyright © 2011 MUSC Foundation for Research Development
```

Comment:
Would it be possible to write a test for feedback_controller#create to ensure that feedback is being created and sent properly in place of this file?